packages/vaadin-text-field/test/text-field.test.js

```diff
@@ -622,4 +622,45 @@ describe('helper slot', () => {
       expect(textField.getAttribute('has-helper')).to.be.equal('slotted');
     });
   });
+
+  describe('nested helper', () => {
+    let field, helper;
+    beforeEach(() => {
+      field = fixtureSync(`
+        <vaadin-text-field label="outer">
+          <vaadin-text-field label="inner" slot="helper">
+            <vaadin-text-field label="inner" slot="helper"></vaadin-text-field>
+          </vaadin-text-field>
+        </vaadin-text-field>
+      `);
+      helper = field.querySelector('[slot="helper"]');
+      field.focus();
+    });
+
+    it('should get focus when clicked', () => {
+      expect(field.hasAttribute('focused')).to.be.true;
+    });
+
+    it('helper should get focus when clicked', () => {
+      helper.inputElement.focus();
+      expect(field.hasAttribute('focused')).to.be.false;
+      expect(helper.hasAttribute('focused')).to.be.true;
+    });
+  });
+
+  describe('text helper', () => {
+    let field, helper;
+    beforeEach(() => {
+      field = fixtureSync(`
+      <vaadin-text-field label='text-field' helper-text='helper-text'>
+      </vaadin-text-field>
+    `);
```

Comment:
Please correct the indentation here:
```suggestion
      field = fixtureSync(`
        <vaadin-text-field label='text-field' helper-text='helper-text'>
        </vaadin-text-field>
      `);
```